### DISCUSSION TITLE
fanatec-dd-pro/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#4] [Ideas]
# fanatec-dd-pro/

If everything seems under control, you’re just not going fast enough. 我的罗技 G29 方向盘套装购买于 2020 年 3 月，如今过去快 4 年了，除了中途因为主板故障导致质保了一个新的机器以外一直稳定工作，期间

https://nova.moe/fanatec-dd-pro/

**Comment by boris1993** [comment]
好巧我也喜欢玩驾驶模拟，用的图马斯特T300法拉利，平时倒不怎么玩ACC这些严肃的竞速游戏，更多是AC逛逛首都高，欧卡2送一下午货，或者地平线5去墨西哥兜风
但是最近又搬家了，新租的房子空间不够，摆不下支架，可能又要忍痛卖掉了 T_T
